src/main/resources/fabric.mod.json

```diff
@@ -32,6 +32,6 @@
     "minecraft": "1.16.x"
   },
   "suggests": {
-    "flamingo": "*"
+    "another-mod": "*"
   }
```

Comment:
I would either keep this as flamingo or remote it all togeather.

I dont think many people use the suggests block, and there are 3 good examples on how to use the depends block so removing it seems fine. As good as flamingo is, its a bit of an in joke for the people that know about it, others it may just confuse.

Reply:
will remove... if only JSON allowed comments, would be a good way to explain some things inline

Reply:
This ended up being merged with `another-mod` instead of `flamingo` or nothing...